library/core/data_callbacks/build.gradle

```diff
@@ -0,0 +1,19 @@
+plugins {
+	id("qsl.module")
+}
+
+qslModule {
+	name = "Quilt Data Callbacks API"
+	moduleName = "data_callbacks"
+	hasMixins = false
+	id = "quilt_data_callbacks"
+	description = "APIs for making events or APIs data-driven"
+	library = "core"
+	moduleDependencies {
+		core {
+			api("qsl_base")
+			testmodOnly("networking")
+			testmodOnly("resource_loader")
```

Comment:
```suggestion
			api("resource_loader")
```

Reply:
Given that resource_loader is actually entirely optional to use this, I'm not sure why this change needs to be made.

Reply:
Uh. Missed this when reading. Though, to realistically use the API resource loader would be needed right?

Reply:
Not necessarily. I think I use it for either surface events or biome modifications without touching the resource loader API, using registry loading events instead

Reply:
Which would still be needed for the modded resources to even be visible to the registry loading events.